features/steps/validation_handling.py

```diff
@@ -203,6 +200,7 @@ def should_apply(items, depth):
 
     # evokes behave error
     generate_error_message(context, [gherkin_outcome for gherkin_outcome in context.gherkin_outcomes if gherkin_outcome.severity in [OutcomeSeverity.WARNING, OutcomeSeverity.ERROR]])
+    pass
```

Comment:
Why this?

Reply:
A debug statement I forgot to remove, sorry 
https://github.com/buildingSMART/ifc-gherkin-rules/commit/4e3ce4409cc08ba0e69ba9f6a9b672f28979fc4b